PROJECT_GOVERNANCE.md

```diff
@@ -0,0 +1,33 @@
+Each project under clj-commons need to have at least a maintainer, and the names of the maintainers should be listed
+ in the `.github/CODEOWNERS` file. The organization as such do not pose any 
+ restrictions on how the individual projects are run, other than what is stated in the 
+ [meta/README](https://github.com/clj-commons/meta/README.md).
+
+  This means that PR review protocol, release cycles, and post-release annoncements are up to the 
+ maintainer of the project, even though we encourage announcing new releases to the main Google Group
+ and the #announcement channel in the Clojurians slack. Also, coding style is up to the maintainers of
```

Comment:
Would probably be interesting to add links to the main Google Group and Clojurians slack. Probably not everyone is aware of those 2 communities (I'm a long time Clojure user and discovered Clojurians slack a few months ago only).